.github/workflows/test.yml

```diff
@@ -212,10 +212,11 @@ jobs:
         version: [02,99]
         include:
           - version: 02
-            compose: "-f process-compose.yaml"
+            test-name: test_native_demo_basic
```

Comment:
so previously this was running smoke and upgrade test both. Now its just upgrade test?

Reply:
nvm I see assert_native_demo_works() being called